docs/dev/ST_ISOVist.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: docs
+title: ST_ISOVist
+category: geom2D/process-geometries
+is_function: true
+description: Compute the visibility from a point
+prev_section: geom2D/process-geometries
+next_section: ST_LineIntersector
+permalink: /docs/dev/ST_ISOVist/
+---
+
+### Signature
+
+{% highlight mysql %}
+GEOMETRY ST_ISOVIST(GEOMETRY point, GEOMETRY obstacles, 
+	DOUBLE maxDistance)
+GEOMETRY ST_ISOVIST(GEOMETRY point, GEOMETRY obstacles, 
+	DOUBLE maxDistance, DOUBLE angleStart, DOUBLE angleStop)
+{% endhighlight %}
+
+### Description
+
+This function compute the visibility polygon obstructed by provided `obstacles`. 
+
+`Obstacles` can be `LINESTRING(S)` ,`POLYGON(S)`  or `GEOMETRYCOLLECTION(S)`. 
+
+Resulting polygon will be enclosed by a circle defined by the maximum distance (`maxDistance`) parameter.
+
+##### Parameters
+
+* `point` : Point coordinates (x, y, (z)) from which the visibility will be calculated,
+* `obstacles` : Geometry or set of geometries grouped into a simple or complex geometry, used as obstacles,
+* `maxDistance` : Only obstacles located within this distance from the `point` are considered in the calculation *(unit depends on your spatial ref)*,
```

Comment:
Sure ??? or only Cartesian computation ?

Reply:
Don't understand. 
This text is extracted / adapted from the source code

Reply:
Yes only Cartesian distance

Reply:
Ok I will add this info